package.json

```diff
@@ -46,6 +46,7 @@
     "enzyme-adapter-react-16": "1.7.1",
     "flux": "^3.1.1",
     "highlight.js": "~9.6.0",
+    "jest-styled-components": "^6.3.1",
```

Comment:
package-lock looks like it has upgraded a lot of stuff - this may need re-generating to be safe